lib/say_cases.rb

```diff
@@ -0,0 +1,59 @@
+class SayCase < OpenStruct
+  def test_name
+    'test_%s' % description.downcase.tr(' ', '_')
+  end
+
+  def workload
+    [
+      "question = '#{input}'",
+      indent(4, assertion),
+    ].join("\n")
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  private
+
+  def indent(size, lines)
+    lines.lines.each_with_object('') { |line, obj| obj << ' ' * size + line }
+  end
+
+  def assertion
+    return error_assertion unless expected
+    return message_assertion if message
```

Comment:
It looks like `message_assertion` isn't being used in this particular exercise.

This check and the method definition can be removed.